abci/cmd/abci-cli/abci-cli.go

```diff
@@ -79,10 +79,11 @@ func RootCmmand(logger log.Logger) *cobra.Command {
 // Structure for data passed to print response.
 type response struct {
 	// generic abci response
-	Data []byte
-	Code uint32
-	Info string
-	Log  string
+	Data   []byte
+	Code   uint32
+	Info   string
+	Log    string
+	Status int32
```

Comment:
Is the size of this field important? If not, consider leaving it as "int"

Reply:
The undelying type of Status in the response of ProcessProposal, as it is [defined](https://github.com/tendermint/tendermint/blob/master/abci/types/types.pb.go#L163) in `types.pb.go`, is `int32`. That's why I defined like that, so we don't need to convert it to another type. I could still change it to `int`, if you consider it's better.